sql/mob_skill_lists.sql

```diff
@@ -644,16 +644,6 @@ INSERT INTO `mob_skill_lists` VALUES ('Gowam',146,40); -- vorpal blade
 -- 147: Humanoid-Galka
 INSERT INTO `mob_skill_lists` VALUES ('Warabouc',148,383); -- tusk
 INSERT INTO `mob_skill_lists` VALUES ('Warabouc',148,385); -- bone_crunch
-INSERT INTO `mob_skill_lists` VALUES ('Humanoid-Hume',149,347);
-INSERT INTO `mob_skill_lists` VALUES ('Humanoid-Hume',149,419);
-INSERT INTO `mob_skill_lists` VALUES ('Humanoid-Hume',149,422);
-INSERT INTO `mob_skill_lists` VALUES ('Humanoid-Hume',149,441);
-INSERT INTO `mob_skill_lists` VALUES ('Humanoid-Hume',149,447);
-INSERT INTO `mob_skill_lists` VALUES ('Humanoid-Hume',149,457);
-INSERT INTO `mob_skill_lists` VALUES ('Humanoid-Hume',149,464);
-INSERT INTO `mob_skill_lists` VALUES ('Humanoid-Hume',149,471);
-INSERT INTO `mob_skill_lists` VALUES ('Humanoid-Hume',149,487);
-INSERT INTO `mob_skill_lists` VALUES ('Humanoid-Hume',149,540);
 INSERT INTO `mob_skill_lists` VALUES ('Humanoid-Hume',149,1390);
```

Comment:
Note to our future selves: Change this list name to something more descriptive, like... 'Tenzen' 😉

Reply:
iI think its a leftover from back when using a ton of checks in the onMobSkillCheck to figure our _which_ mob it was, was being allowed instead of just making a new list.